test/chain.ts

```diff
@@ -15,6 +15,23 @@ describe('chain codes', function() {
     Codes.internal.p2wsh,
   ];
 
+  const purposeList = [
```

Comment:
I'm confused here: Purposes are ['internal', 'external']. Why are we listing the scriptTypes in a list that is named "purposeList"?

Reply:
Changed the name to `purposeByScriptTypeList`